### PR TITLE
Add Kirigami Stealth Bumper

### PR DESCRIPTION
**Pull request by MapleLeafMakers**
### Which mods are added by this PR?

Kirigami Stealth Bumper

### How was it tested?

Installed and in use on my own V0 and many others.

The design was also adapted by LDO for use with their kirigami LED PCB, and that version is WIDELY used.

### ⚠️⚠️**PLEASE READ AND COMPLETE THE CHECKLIST BELOW**⚠️⚠️

  * [X] I have read the rules available [here](https://github.com/VoronDesign/VoronUsers/wiki/Mod-Submission-Rules) and
        my mod adheres to these rules.
  * [X] This mod was created by myself and I agree to publish it under the repository
        [license](https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md)

### CODE REVIEW

**Comment by MediumRare-david** [comment]
All looks good to me!

**Review by MediumRare-david — APPROVED** [review]
looks great. Awesome mod!

**Comment by telxoid** [comment]
Looks like you may be missing some chamfers on the build plate.

File: printer_mods/MapleLeafMakers/Kirigami_Stealth_Bumper/STL/neopixel_button_carrier.stl
Comment (MediumRare-david) [inline printer_mods/MapleLeafMakers/Kirigami_Stealth_Bumper/STL/neopixel_button_carrier.stl:1]: Possibly needs a chamfer on the bottom to compensate for elephants foot.

**Comment by MapleLeafMakers** [comment]
I added a 0.4mm bottom chamfer to the bottom outside edge of neopixel_button_carrier and rainbow_barf_carrier.  The inner hole only had space for a 0.2mm chamfer.  

I tried adding it to the logo_insert in CAD, but it impacts the aesthetics of the logo too much, I'd like to keep that as-is.

**Review by MediumRare-david — APPROVED** [review]
thanks for adding those chamfers

**Comment by telxoid** [comment]
All good, lets push it forward.

**Pull request merged**